console/src/main/scripts/gulpfile.js

```diff
@@ -152,11 +160,11 @@ var gulpTsc = function(done) {
     });
 };
 
-gulp.task('tsc', ['clean-defs'], function (done) {
+gulp.task('tsc', ['clean-defs', 'tsd'], function (done) {
   gulpTsc(done);
 });
 
-gulp.task('tsc-live', ['copy-sources','clean-defs'], function (done) {
+gulp.task('tsc-live', ['copy-sources','clean-defs', 'tsd'], function (done) {
```

Comment:
I'm not sure that this needs to run with every change. The type definition files shouldnt change much and it slows the compiling down to have to hit those urls and pull in that data. Right now there is only one type def file and I can see the slowdown. With 10+ type def files its a real tax. Probably even too much for Travis builds (especially given the static nature of those files it isn't necessary). 
This is a great task to have, just think it should be run manually (and type defs checked in) not automatically.